notebooks/COS/DataDl/DataDl.ipynb

```diff
@@ -160,7 +160,7 @@
     "\n",
```

Comment:
<p>Comments included in this section may be redundant, as the markdown cells above already explain what each import is for.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>